### ISSUE TITLE
Inline closure with use statement tokenized as T_FUNCTION instead of T_CLOSURE

### ISSUE THREAD

**Issue by pfrenssen** [squizlabs/PHP_CodeSniffer#472]
The closure in the following code snippet is wrongly tokenized as being a `T_FUNCTION` where it should be a `T_CLOSURE`:

```
array_reduce($array, function ($carry, $item) use ($variable) {
  return TRUE;
});
```

**Comment by gsherwood** [comment]
I'm not seeing this problem. The code I am testing is:
```php
<?php
array_reduce($array, function ($carry, $item) use ($variable) {
  return TRUE;
});
```
The output I get from `phpcs -vv temp.php` is:
```
Processing temp.php
	*** START PHP TOKENIZING ***
	Process token [0]: T_OPEN_TAG => <?php\n
	Process token [1]: T_STRING => array_reduce
	Process token  2 : T_OPEN_PARENTHESIS => (
	Process token [3]: T_VARIABLE => $array
	Process token  4 : T_COMMA => ,
	Process token [5]: T_WHITESPACE => ·
	Process token [6]: T_FUNCTION => function

** snip **

	*** START ADDITIONAL PHP PROCESSING ***
	* token 6 on line 2 changed from T_FUNCTION to T_CLOSURE
		* cleaned 22 (T_WHITESPACE) *
		* cleaned 23 (T_WHITESPACE) *
		* cleaned 24 (T_RETURN) *
		* cleaned 25 (T_WHITESPACE) *
		* cleaned 26 (T_TRUE) *
		* cleaned 27 (T_SEMICOLON) *
		* cleaned 28 (T_WHITESPACE) *
	*** END ADDITIONAL PHP PROCESSING ***
[PHP => 32 tokens in 4 lines]...
DONE in 2ms (6 errors, 0 warnings)
```
Can you give me any more information about to replicate the problem? Is a specific sniff causing an issue? Is there a larger piece of code that is causing the problem?

**Comment by pfrenssen** [comment]
I'm sorry for wasting your time, I didn't realize I was still on an experimental branch where I made a change that placed the T_USE token in the group of scopeOpeners in order to investigate a different issue.

**Comment by gsherwood** [comment]
No problem. Thanks for getting back to me.
